frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -153,6 +153,7 @@ end
 function ReaderRolling:onReaderReady()
     self:setupTouchZones()
     self.setupXpointer()
+    self.old_page = 0
```

Comment:
`self.old_page` should be `0`.
`0` means that the number of pages in the document has not yet been generated.
Previously was `1` but it's possible that document has one page.